plugins/net_plugin/net_plugin.cpp

```diff
@@ -2206,12 +2213,15 @@ namespace eosio {
 
     my->sync_master.reset( new sync_manager( def_sync_rec_span ) );
 
-    my->connector_period = def_conn_retry_wait;
+    my->connector_period = options.count( "connection-cleanup-period") ? std::chrono::seconds(options.at("connection-cleanup-period").as<int>()) : def_conn_retry_wait;
```

Comment:
Since you specify a default this will always be true. So if not specified it will be 30, NOT def_conn_retry_wait.